core-ktx/src/main/kotlin/Context.kt

```diff
@@ -0,0 +1,22 @@
+package com.redmadrobot.extensions.core
+
+import android.app.Activity
+import android.content.Context
+import android.content.ContextWrapper
+import android.view.Window
+
+public tailrec fun Context.findActivity(): Activity? {
+    return when (this) {
+        is Activity -> this
+        is ContextWrapper -> baseContext.findActivity()
+        else -> null
+    }
+}
+
+public tailrec fun Context.findWindow(): Window? {
```

Comment:
It would be great to add KDoc comments here. And to mention this change in changelog.